### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**
https://github.com/paritytech/polkadot/pull/7149 on release branch

### CODE REVIEW

File: runtime/westend/src/weights/runtime_parachains_hrmp.rs [hunk 1]
```diff
@@ -129,23 +131,23 @@ impl<T: frame_system::Config> runtime_parachains::hrmp::WeightInfo for WeightInf
...
-		Weight::from_parts(1_097_373_000, 0)
-			.saturating_add(Weight::from_parts(0, 8850))
-			// Standard Error: 97_518
-			.saturating_add(Weight::from_parts(3_170_650, 0).saturating_mul(i.into()))
-			// Standard Error: 97_518
-			.saturating_add(Weight::from_parts(3_304_346, 0).saturating_mul(e.into()))
+		//  Measured:  `158 + e * (100 ±0) + i * (100 ±0)`
+		//  Estimated: `3620 + e * (2575 ±0) + i * (2575 ±0)`
+		// Minimum execution time: 1_068_293_000 picoseconds.
+		Weight::from_parts(2_873_264, 0)
```
Comment (EgorPopelyaev) [inline runtime/westend/src/weights/runtime_parachains_hrmp.rs:137]: Huge jump down

**Review by EgorPopelyaev — APPROVED** [review]

File: runtime/kusama/constants/src/weights/block_weights.rs [hunk 2]
```diff
@@ -42,17 +43,17 @@ parameter_types! {
...
 	/// Percentiles nanoseconds:
-	///   99th: 6_991_352
-	///   95th: 6_933_543
-	///   75th: 6_854_332
+	///   99th: 3_600_149
+	///   95th: 3_542_240
+	///   75th: 3_479_155
 	pub const BlockExecutionWeight: Weight =
-		Weight::from_parts(WEIGHT_REF_TIME_PER_NANOS.saturating_mul(6_818_965), 0);
+		Weight::from_parts(WEIGHT_REF_TIME_PER_NANOS.saturating_mul(3_467_068), 0);
```
Comment (coderobe) [inline runtime/kusama/constants/src/weights/block_weights.rs:56]: block execution weights halved

**Comment by coderobe** [comment]
@s3krit looks like your thing didn't trigger

**Review by chevdor — APPROVED** [review]

**Comment by chevdor** [comment]
https://weights.tasty.limo/compare?old=refs/tags/v0.9.41&new=367ccaa1ee3c34b0c64999a08cdddb900519a457&threshold=90&unit=weight&repo=polkadot&path_pattern=runtime%2F**%2Fsrc%2Fweights%2F**%2F*.rs&method=guess-worst&ignore_errors=true&git_pull=false

**Comment by s3krit** [comment]
> @s3krit looks like your thing didn't trigger

Looking into it, cheers

**Pull request merged**